src/transformers/models/auto/modeling_auto.py

```diff
@@ -500,6 +502,7 @@
     [
         # Model for Seq2Seq Causal LM mapping
         (M2M100Config, M2M100ForConditionalGeneration),
+        (Speech2TextConfig, Speech2TextForConditionalGeneration),
```

Comment:
I think I'd prefer to not add it here since it's not really a Seq2Seq LM but a Seq2Seq ASR model